MapboxNavigation/SpeechSynthesizersController.swift

```diff
@@ -0,0 +1,116 @@
+
+import AVFoundation
+import MapboxDirections
+import MapboxCoreNavigation
+import MapboxSpeech
+
+/// `SpeechSynthesizing`implementation, aggregating other implementations, to allow 'fallback' mechanism.
+/// Can be initialized with array of synthesizers which will be called in order of appearance, untill one of them is capable to vocalize current `SpokenInstruction`
+open class SpeechSynthesizersController: SpeechSynthesizing {
+    
+    // MARK: - Properties
+    
+    public var delegate: SpeechSynthesizingDelegate?
+    
+    public var muted: Bool = false {
+        didSet {
+            speechSynthesizers.forEach { $0.muted = muted }
+        }
+    }
+    public var volume: Float = 1.0 {
+        didSet {
+            speechSynthesizers.forEach { $0.volume = volume }
+        }
+    }
+    public var isSpeaking: Bool {
+        return speechSynthesizers.first(where: { $0.isSpeaking }) != nil
+    }
+    public var locale: Locale = Locale.autoupdatingCurrent {
+        didSet {
+            speechSynthesizers.forEach { $0.locale = locale }
+        }
+    }
+    
+    public  let speechSynthesizers: [SpeechSynthesizing]
+    
+    private var currentLegProgress: RouteLegProgress?
+    
+    // MARK: - Lifecycle
+    
+    public init(_ speechSynthesizers: [SpeechSynthesizing]? = nil, accessToken: String? = nil) {
```

Comment:
The access token (and host) is a detail specific to MapboxSpeechSynthesizer. On master, MapboxVoiceController leaves the default SpeechController’s access token blank, but NavigationViewController ends up creating a different default SpeechSynthesizer with a matching access token, so it isn’t really even necessary for MapboxVoiceController to provide a default SpeechController.

Here, we could make `speechSynthesizers` required and remove the `accessToken` argument; then callers (still within the SDK) would need to create the appropriate MapboxSpeechSynthesizer and SystemSpeechSynthesizer with the context they have on hand.